lib/logger/index.ts

```diff
@@ -137,6 +125,9 @@ export /* istanbul ignore next */ function addStream(
 
 export function levels(name: string, level: bunyan.LogLevel): void {
   bunyanLogger.levels(name, level);
+  if (level === 'trace') {
+    inject({ formatter: new RenovateFormatter(logger) });
```

Comment:
we should call this on early startup from `lib/renovate.ts`.

```ts
const njstrace = inject({ formatter: new RenovateFormatter(logger), enabled: false });
```
```suggestion
   njstrace.enabled = true;
```
https://www.npmjs.com/package/njstrace#enabled-boolean

and mock it globally in `test/globals`